js/ajaxTask.js

```diff
@@ -0,0 +1,34 @@
+function send() {
+    let username = getElementByIdFromDocument("username").value;
+    let password = getElementByIdFromDocument("password").value;
+    let confirmPassword = getElementByIdFromDocument("confirmPassword").value;
+
+    let object = { username, password, confirmPassword };
+    let jsonToBeSent = converObjectToJSON(object);
+
+    const url = "./register.php"
+    let callback = function (text) {
+        console.log(text);
+    };
+
+    ajax(url, { success: callback, data: jsonToBeSent });
+}
+function getElementByIdFromDocument(elementName) {
+    return document.getElementById(elementName);
+}
+function converObjectToJSON(object) {
+    return JSON.stringify(object);
+}
+function ajax(url, settings) {
+    var xhr = new XMLHttpRequest();
+    xhr.onload = function () {
+        if (xhr.status == 200) {
+            settings.success(xhr.response);
+        } else {
+            console.error(xhr.response);
```

Comment:
You may want to notify the user in some way about this - maybe in the content of the html page.